src/js/mqtt.js

```diff
@@ -332,7 +296,8 @@ function getClient(connectOptions) {
       connectOptions.message = new Buffer(connectOptions.message.toString());
     }
   }
-  return new MQTTClient(connectOptions);
+
+  return new MQTTClient(connectOptions, callback);
 }
 
-exports.getClient = getClient;
+exports.connect = connect;
```

Comment:
This changes the API, thus the documentation should also be updated.

Reply:
Done.